backend/app/controllers/api/v1/users_controller.rb

```diff
@@ -4,6 +4,35 @@ module Api
   module V1
     class UsersController < ApplicationController
       include Dry::Monads::Result::Mixin
+      before_action :set_user
+
+      def show
+        user = set_user
```

Comment:
We don't need to call this, because we're using `before_action`.
Just use user instance variable directly (or via private attr_reader even better)